File: bubble.py
import time

def bubbleSort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1] :
                arr[j], arr[j+1] = arr[j+1], arr[j]

while (True):
    arr = []
    line = input()
    if (not line):
        break
    line = line.split()
    for i in line:
        arr.append(int(i))
    inicio = time.time()
    bubbleSort(arr)
    fin = time.time()

    print(fin-inicio)

File: bubble.js
function bubbleSort( arr, n){
    for (var i = 0; i < n-1; i++){
        for (var j = 0; j < n-i-1; j++){
            if (arr[j] > arr[j+1]){
                var aux = arr[j+1];
                arr[j+1] = arr[j];
                arr[j] = aux;
            }
        }
    }
}